### ISSUE TITLE
Policy after controller action

### ISSUE THREAD

**Issue by Notsew** [balderdashy/sails#2195]
Is it possible to have some policies run after the controller action, similar to Rails after_filter method?

**Comment by dirwin517** [comment]
:+1:

**Comment by sailsbot** [comment]
Thanks for posting, @Notsew.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by luqezman** [comment]
News?

**Comment by outofthisworld** [comment]
I would like this too :))

**Comment by luqezman** [comment]
@outofthisworld sailsjs dont have some dispatcher events :'(

Example: before policies be called, after policies be called.

It is really hard to intercept and implement somethings :'(

I tried to inject middleware but inside of middleware, we dont have req.options and it is undefined :'(
